src/current/v25.2/changefeed-message-envelopes.md

```diff
@@ -0,0 +1,928 @@
+
+You can use changefeed [options](#option-reference) to customize message contents in order to integrate with your downstream requirements. For example, the previous state of the row, the origin cluster's metadata, or the schema of the event payload in the message. Envelope configuration also allows you to prioritize change event metadata versus changefeed throughput.
+
+The possible envelope fields support use cases such as:
+
+- [Auditing](#audit-changes-in-data) changes in data.
+- Enabling [full-fidelity changefeed messages](#enable-full-fidelity-message-envelopes).
+- Routing events based on [operation type](#route-events-based-on-operation-type).
+- Automatically [generating or synchronizing schemas](#add-envelope-schema-fields) in downstream consumers.
+
+This page covers:
+
+- [Overview](#overview) for a brief summary of the CockroachDB changefeed envelope fields.
+- [Use case](#use-cases) examples.
+- Reference lists:
+    - The [options](#option-reference) to configure the envelope.
+    - The supported envelope [fields](#field-reference).
+
+{{site.data.alerts.callout_info}}
+You can also specify the _format_ of changefeed messages, such as Avro. For more details, refer to [Message formats]({% link {{ page.version.version }}/changefeed-messages.md %}#message-formats).
+{{site.data.alerts.end}}
+
+## Overview
+
+The envelope of a changefeed message depends on the following:
+
+- The [configured envelope and changefeed options](#option-reference)
+- The [sink]({% link {{ page.version.version }}/changefeed-sinks.md %})
+- The [message format]({% link {{ page.version.version }}/changefeed-messages.md %}#message-formats)
+
+The envelope can contain any of the outlined fields in this section. For more details, refer to the complete reference lists that explain each [envelope field](#field-reference) and [configurable option](#option-reference) to build a changefeed's envelope structure.
+
+The default [`wrapped` envelope](#wrapped) will be similar in structure to:
+
+~~~json
+{
+  "after": {
+    "after": { 
+      "col_a": "new_value",
+      // ... other columns ...
+    },
+  },
+  "key": [
+    // primary key for the row
+  ]
+}
+~~~
+
+You can optionally include the state of the row [before and after](#audit-changes-in-data) the change event and [operation](#route-events-based-on-operation-type), [origin](#preserve-the-origin-of-data), and [timestamp metadata](#updated) about the change. It is important to consider that increasing the size of each message can have an impact on changefeed throughput.
+
+The payload containing the change event's table data and metadata for the event:
+
+~~~json
+{
+  "payload": { 
```

Comment:
- now we're demoing an enriched envelope? the text didnt say that. i thought we were still talking about wrapped.
- payload wrapper only present if schema is

Reply:
this is still a problem i think. you switch envelope types without mentioning it.

Reply:
@asg0451 Latest commit removes the entire Overview section. This was just meant to be a high-level of the envelope features before the use case section, but mostly duplicates information that is throughout this page. Because this isn't landing, I've removed and moved the links to the content on the page into the introductory paragraphs. This should resolve the issue here.